### PR TITLE
Added tests to update repo command

### PR DESCRIPTION
**Pull request by brunats**
**- What I did**
Hi Ritchiers,
Add tests for the rit update repo command

> Master
> master
> ->
> PR
>pr

**- How to verify it**
> Clone this pr with the command
```bash
git fetch upstream pull/515/head:improvement/added-tests-to-update-repo-command

```
> And run goimports
```bash
git checkout master
make build && cp dist/linux/rit /usr/local/bin/
go test -v -failfast -short -coverprofile=bin/cov.out ./pkg/cmd && go tool cover -func=bin/cov.out # Run unit tests of the cmd pkg

git checkout improvement/added-tests-to-update-repo-command
make build && cp dist/linux/rit /usr/local/bin/
go test -v -failfast -short -coverprofile=bin/cov.out ./pkg/cmd && go tool cover -func=bin/cov.out #  Run and see the differences
```

**- Description for the changelog**

Added tests to update repo command

### CODE REVIEW

File: pkg/cmd/update_repo.go [hunk 1]
```diff
@@ -127,7 +131,7 @@ func (up updateRepoCmd) runStdin() CommandRunnerFunc {
 	return func(cmd *cobra.Command, args []string) error {
 		r := formula.Repo{}
 
-		err := stdin.ReadJson(os.Stdin, &r)
+		err := stdin.ReadJson(cmd.InOrStdin(), &r)
 		if err != nil {
 			return err
 		}
```
Comment (victor-schumacher) [inline pkg/cmd/update_repo.go:137]: can u minimize the err check?

File: pkg/cmd/update_repo_test.go [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+					},
+				},
+				inList: inputListCustomMock{
+					list: func(name string, items []string) (string, error) {
+						if name == questionSelectARepo {
+							return "someRepo1", nil
+						}
+						if name == questionAVersion {
+							return "1.0.0", nil
+						}
```
Comment (victor-schumacher) [inline pkg/cmd/update_repo_test.go:75]: this is a way to send different inputs when you have more than one list? 
awesome :sunglasses:

**Review by victor-schumacher — APPROVED** [review]

File: pkg/cmd/update_repo.go (hunk 1 above)
Comment (brunasilvazup) [reply pkg/cmd/update_repo.go:137]: Oops, yes I can, I'll change it.

File: pkg/cmd/update_repo_test.go (hunk 2 above)
Comment (brunasilvazup) [reply pkg/cmd/update_repo_test.go:75]: Yes, it is for any type of function that has an 'index'
(:

File: pkg/cmd/update_repo_test.go [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+	"strings"
+	"testing"
+
+	"github.com/ZupIT/ritchie-cli/pkg/formula"
+	"github.com/ZupIT/ritchie-cli/pkg/git"
+	"github.com/ZupIT/ritchie-cli/pkg/git/github"
+	"github.com/ZupIT/ritchie-cli/pkg/prompt"
+)
+
+func Test_NewUpdateRepoCmd(t *testing.T) {
```
Comment (lucasdittrichzup) [inline pkg/cmd/update_repo_test.go:34]: this is out of the standard, right?

Comment (brunasilvazup) [reply pkg/cmd/update_repo_test.go]: I agree, changed to the Golang standard.

**Review by kaduartur — APPROVED** [review]

**Pull request merged**